app/controllers/admin/user_sessions_controller.rb

```diff
@@ -1,5 +1,6 @@
 class Admin::UserSessionsController < Admin::AdminController
-  skip_before_filter :require_admin_and_check_for_password_change
+  skip_before_filter :require_admin, only: [:new, :create]
+  skip_before_filter :check_for_password_change
```

Comment:
This doesn't make much sense to me. It seems like instead of checking if the user needs to change their password on each login, which would seem "Usual (tm)" to me, we're checking on each pageview *except* when a user creates a session.

Perhaps I'm missing a requirement?